jkube-kit/config/service/src/test/java/org/eclipse/jkube/kit/config/service/openshift/ImageStreamServiceTest.java

```diff
@@ -245,12 +246,12 @@ void appendImageStreamResource_whenMultipleTagsInImageStreamStatus_thenAddLatest
         assertThat(kubernetesList)
             .isNotNull()
             .extracting(DefaultKubernetesResourceList::getItems)
-            .asList()
+            .asInstanceOf(InstanceOfAssertFactories.list(DefaultKubernetesResourceList.class))
```

Comment:
This doesn't look correct. You need to check the return value type for method being extracted in previous `extracting` call, in this case it's `HasMetadata.class`